Makefile.include

```diff
@@ -166,6 +170,12 @@ endif
 	$(AD)$(OBJCOPY) $(OFLAGS) $(ELFFILE) $(HEXFILE)
 endif
 
+..toolchain-check:
```

Comment:
The reasoning was, that if the compiler exists, probably the toolchain is installed, too. However, since I introduced checks for other tools (like gdb or objdump as well), it makes probably sense to rename. I only wonder if we should also check for the recommended toolchain version somehow?